Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -691,6 +687,9 @@ static void getAllScriptsInPDFDocument(CGPDFDocumentRef pdfDocument, Vector<Reta
     [m_containerLayer addSublayer:m_scrollCornerLayer.get()];
     if ([m_pdfLayerController respondsToSelector:@selector(setDeviceColorSpace:)])
         [m_pdfLayerController setDeviceColorSpace:screenColorSpace(m_frame->coreFrame()->view()).platformColorSpace()];
+    
+    if ([getPDFLayerControllerClass() respondsToSelector:@selector(setUseIOSurfaceForTiles:)])
+        [getPDFLayerControllerClass() setUseIOSurfaceForTiles:false];
```

Comment:
Just to confirm, this also takes care of `PDFKit2_UseWhippet` right?

Reply:
Yep per the comments on the property: // Disabling it also disables rendering with Whippet.

Reply:
`false` should be `NO` to be more idiomatic Objective-C.